inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -369,7 +338,7 @@
                         "type": "string"
                     },
                     "cnum": {
-                        "type": "array"
+                        "type": "string"
```

Comment:
Ah yes, I was looking at the references statistics. And yes this is still strange.